lsp-inline-completion.el

```diff
@@ -202,40 +202,36 @@ text range that was updated by the completion"
   (-let* ((suggestion
            (elt lsp-inline-completion--items
                 lsp-inline-completion--current))
-          ((&InlineCompletionItem? :insert-text :range?) suggestion)
-          ((&RangeToPoint :start :end) range?)
+          ((&InlineCompletionItem? :insert-text :range? (&RangeToPoint :start :end)) suggestion)
           (start-point (or start (point)))
           (showing-at-eol (save-excursion
                             (goto-char start-point)
-                            (eolp)))
+                            (and (not (bolp)) (eolp))))
```

Comment:
If I recall correctly, I had to handle this specifically because the overlay did not display at the right place when EOL. In this case, I had display the overlay at the character before eol (I believe this was a hint I took from the company preview frontend).

I don't remember seeing the same issue on `bol` (and have not been able to get copilot to answer anything at bol, so I can't test it), so I'm curious -- why shift the overlay display point when bol?